src/vectorclock.erl

```diff
@@ -26,38 +26,84 @@
 -endif.
 
 -export([
-         get_clock_of_dc/2,
-         set_clock_of_dc/3,
-         from_list/1,
-         new/0,
-         eq/2,
-         lt/2,
-         gt/2,
-         le/2,
-         ge/2,
-         strict_ge/2,
-         strict_le/2,
-         max/1,
-         min/1,
-	 to_json/1,
-	 from_json/1]).
+    get_clock_of_dc/2,
+    set_clock_of_dc/3,
+%%    create_commit_vector_clock/3,
+    from_list/1,
+    from_dict/1,
+    to_dict/1,
+    size/1,
+    new/0,
+    eq/2,
+    lt/2,
+    gt/2,
+    le/2,
+    ge/2,
+    find/2,
+    strict_ge/2,
+    strict_le/2,
+    max/1,
+    min/1,
+    fold/3,
+    store/3,
+    fetch/2,
+    erase/2,
+    map/2,
+    max_vc/2,
+    %%    merge/3,
+    min_vc/2, to_json/1, from_json/1]).
 
 -export_type([vectorclock/0]).
 
 -spec new() -> vectorclock().
 new() ->
     dict:new().
-          
+
+to_dict(VC) -> VC.
+%%    dict:from_list(VC).
+
+from_dict(Dict) -> Dict.
+%%    from_list(dict:to_list(Dict)).
+
+size(VC) ->
+    dict:size(VC).
+
+fold(F, Acc, [{Key,Val}|D]) ->
```

Comment:
I removed the fold function. It is not used. I had used it at a point where I was thinking of changing the vectorclock to be an orddict (which is a list)